### ISSUE TITLE
AssertionError on save_submission

### ISSUE THREAD

**Issue by chengchingwen** [YichenGong/Densely-Interactive-Inference-Network#7]
https://github.com/YichenGong/Densely-Interactive-Inference-Network/blob/b36b9220dc16084240bebed51c575d43a8529f12/python/train_mnli.py#L478
`save_submission(path, IDs, logits[1:])` should be just `save_submission(path, IDs, logits)`

As

https://github.com/YichenGong/Densely-Interactive-Inference-Network/blob/b36b9220dc16084240bebed51c575d43a8529f12/python/train_mnli.py#L477

have sliced logits already
